src/tools/illink/src/ILLink.RoslynAnalyzer/TrimAnalysis/TrimAnalysisReflectionAccessPattern.cs

```diff
@@ -34,23 +35,23 @@ namespace ILLink.RoslynAnalyzer.TrimAnalysis
 			OwningSymbol = owningSymbol;
 		}
 
-		public TrimAnalysisReflectionAccessPattern Merge (ValueSetLattice<SingleValue> lattice, TrimAnalysisReflectionAccessPattern other)
-		{
-			Debug.Assert (Operation == other.Operation);
-			Debug.Assert (SymbolEqualityComparer.Default.Equals (ReferencedMethod, other.ReferencedMethod));
-
-			return new TrimAnalysisReflectionAccessPattern (
-				ReferencedMethod,
-				lattice.Meet (Instance, other.Instance),
```

Comment:
The Instance property doesn't seem to be used - we should remove it. It was the sole reason for having a Merge... without it there's really no reason to have Merge.